staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2446,6 +2446,22 @@ type Container struct {
 	// +optional
 	// +listType=atomic
 	ResizePolicy []ContainerResizePolicy `json:"resizePolicy,omitempty" protobuf:"bytes,23,rep,name=resizePolicy"`
+	// Restart policy for the container.
+	// This may only be set for init containers.
+	// The only supported value is "Always".
```

Comment:
This is confusing when stated this way, implying the old behavior is gone.

Instead, start by saying “if this value is not present, <how the existing function works>.  The only allowed value is Always, <describe that tersely>.  <more detail about that value>”

We want to define the default behavior first (unset), then allowed values, then any nuance.

Reply:
Fixed